Beis.LearningPlatform.Web/Models/CmsNextPageRadioButtonViewModel.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Beis.LearningPlatform.Web.StrapiApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Beis.LearningPlatform.Web.Models
+{
+    public class CmsNextPageRadioButtonViewModel
+    {
+        private readonly CMSPageComponent _cmsPageComponent;
+
+        public CmsNextPageRadioButtonViewModel(CMSPageComponent cmsPageComponent)
+        {
+            _cmsPageComponent = cmsPageComponent;
+        }
+
+        public bool HasContent
+        {
+            get
+            {
+                return !string.IsNullOrEmpty(_cmsPageComponent.Radio1Text)
```

Comment:
Is Whitespace allowed?

Reply:
Don't know. Let me know asap. Need to get this out in this sprint

Reply:
I can't say. If the whitespace is not allowed then please change it to string.IsNullOrWhiteSpace where ever there is NullOrEmpty